frontend/src/metabase/query_builder/actions/core/initializeQB.unit.spec.ts

```diff
@@ -114,9 +113,7 @@ async function setup({
   const card = question.card();
 
   if ("id" in card) {
-    xhrMock.get(`/api/card/${card.id}`, {
-      body: JSON.stringify(card),
-    });
+    nock(/.*/).get(`/api/card/${card.id}`).reply(200, card);
```

Comment:
Wondering if we should do something about a constant need to do `nock(/.*/)`? Maybe add a tiny wrapper to `__support__`? For instance:

```jsx
// __support__/http.ts

import _nock from "nock"

export const nock = () => _nock(/.*/);

// initializeQb.unit.spec.ts

import { nock } from "__support__/http"

nock().get(`/api/card/${card.id}`).reply(200, card);
```